explorer/src/providers/accounts/rewards.tsx

```diff
@@ -8,15 +8,19 @@ import { reportError } from "utils/sentry";
 
 const PAGE_SIZE = 15;
 
+export type NoReward = {
+  epoch: number;
```

Comment:
What's the purpose of this over using `null`? Doesn't look like `epoch` is used anywhere

Reply:
This was used in the reconciler: https://github.com/solana-labs/solana/blob/master/explorer/src/providers/accounts/rewards.tsx#L42-L43 but I think I'll go back to using null instead.